main.js

```diff
@@ -138,9 +139,7 @@ function saveOutfit() {
   clearInputField(saveOutfit);
 }
 
-window.addEventListener('load', getOutfits);
-
-function getOutfits(){
+function getOutfitCards(){
   for (var i = 0; i < localStorage.length; i++){
     savedOutfitsList.insertAdjacentHTML('beforeend', (localStorage.getItem(localStorage.key(i))));
   }
```

Comment:
This function loops through the keys saved in local storage and adds the value of each (I stored the HTML as the value) to the savedOutfitsList. It is invoked on load (see the event listener I added above).
A later card deals with removing items and making sure they don't reappear on reload, so I left that "bug" alone for now.

Reply:
Nice job! This looks awesome. I'll complete this merge to master. I think eventually we will need to store the actual outfit object to local storage instead of the html. That will allow us to re-dress the bear with the correct outfit items. But that should be easy enough to work into what you already built. Nice!